tests/AbstractLoopTest.php

```diff
@@ -534,12 +534,12 @@ public function testSignalsKeepTheLoopRunning()
     {
         $function = function () {};
         $this->loop->addSignal(SIGUSR1, $function);
-        $this->loop->addTimer(1.5, function () use ($function) {
+        $this->loop->addTimer(2, function () use ($function) {
             $this->loop->removeSignal(SIGUSR1, $function);
             $this->loop->stop();
         });
 
-        $this->assertRunSlowerThan(1.5);
+        $this->assertRunSlowerThan(2);
```

Comment:
What is the motivation for this change?

Reply:
I thought that there was a problem with floating point accuracy, but it was not. It should be changed to the previous `1.5` value. Fixed.